specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/costmanagement.json

```diff
@@ -1178,7 +1178,7 @@
           "ReservedInstances"
         ],
         "operationId": "GenerateReservationDetailsReport_ByBillingProfileId",
-        "description": "Generates the reservations details report for provided date range asynchronously by billing profile.",
+        "description": "Generates the reservations details report for provided date range asynchronously by billing profile. The Reservation usage details can be viewed by only certain enterprise roles by default. For more details on the roles see, https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/understand-ea-roles#usage-and-costs-access-by-role",
```

Comment:
The link is taking to EA roles - do we have one for GTM roles

Reply:
updated